src/main/java/frc/robot/subsystems/Wrist.java

```diff
@@ -21,6 +23,8 @@ public class Wrist extends SubsystemBase {
   private double highLim;
   private double lowLim;
 
+  private double stow;
```

Comment:
Is this used anywhere? you could make a `stowCommand` method that calls `setToTarget(stow)` and then we change the stow position in this class instead of in `CompetitionRobotContainer`